sheetstorm/management/commands/data_load.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from users.factories import UserFactory
+
+User = get_user_model()
```

Comment:
A tiny detail, but I would suggest adding another blank line to visually separate imports from constants definition.

Reply:
I tried to do that, but Black removed this blank line.